docs/content/test/TestMinimized.md

```diff
@@ -50,9 +50,11 @@ If this is your case, add these resources to `res/raw/keep.xml` like this:
            tools:keep="@layout/page_object*"/>
 ```
 
-### Try latest stable version of R8
+### X already has a mapping
 
-Some issues probably solved in new version of r8. For example: ["already has a mapping" one](https://issuetracker.google.com/issues/122924648)
+[Issue](https://issuetracker.google.com/issues/122924648)
```

Comment:
I'd prefer to see an issue number in the link:
`[Issue #122924648](https://issuetracker.google.com/issues/122924648)`